sources/packages/web/src/components/institutions/modals/EditInstitutionUserModal.vue

```diff
@@ -31,6 +31,17 @@
               </span>
               <InputSwitch v-model="isAdmin" />
             </v-col>
+            <v-col v-if="isAdmin">
+              <span class="form-text text-muted mb-2"
+                ><strong>Select Admin Role</strong></span
+              >
+              <Dropdown
+                v-model="selectedAdminRole"
+                :options="adminRoles"
+                :style="{ width: '20vw' }"
```

Comment:
please avoid inline style

Reply:
As it is Primevue as discussed ignoring writing css class part. Instead added a to-do comment.